visual.js

```diff
@@ -36,9 +36,10 @@ var vis = {
     },
     getGiveawayData: function (item) {
         var enableIncrement = true;
-        var image = $('a.giveaway_image_thumbnail', item);
-        if ($('.giveaway_image_thumbnail,.giveaway_image_thumbnail_missing', item).length == 0) image = null;
-        else image.attr('style').replace('background-image:url(', '').replace(');', '');
+        var image = $('a.giveaway_image_thumbnail,.giveaway_image_thumbnail_missing', item);
+        if (image.length == 0) image = null;
+        // What does this do?
+        //else image.attr('style').replace('background-image:url(', '').replace(');', '');

```

Comment:
oh ok, this line (41/42) originally set image to the background image URL. I didn't see the original code before committing